apps/researcher/src/app/[locale]/(objects)/objects/[id]/metadata.tsx

```diff
@@ -105,14 +105,24 @@ export async function MetadataEntry({
   const t = useTranslations('ObjectDetails');
   const format = await getFormatter();
 
+  if (isCurrentPublisher && !children) {
```

Comment:
This is moved to line 36. Can you remove this block and add the classnames to the code in line 36?